### ISSUE TITLE
[INFRA - Social Media] Create Hootsuite Account

### ISSUE THREAD

**Issue by TetianaFTV** [microprofile/microprofile-marketing#487]
On MP marketing call (April 19) it was decided to get 1 user Hootsuite account for automation of social media posts sharing. 

**For this we need:**
- create MP gmail email, e.g. microprofile-media@tbd.com
- register Hootsuite Professional account with this newly created email at https://www.hootsuite.com/plans/professional
- assign payment method within this account (at the beginning there will be 30 days trial)
- connect social media accounts
- share credentials with the MP members who are going to be involved in scheduling the posts

@aeiras adding you here to proceed with the next steps

**Comment by aeiras** [comment]
Thank you, Tetiana! 

@rstjames can you please look beyond the email infra, requirements and tell us what else is needed to get the account set up?

Thank you for the leg-work!

**Comment by aeiras** [comment]
Tetiana, related to this ticket, https://github.com/eclipse/microprofile-marketing/issues/499. 
new ticket was needed b/c it creates an INFRA email that will be used not only for Hootsuite but also as needed.

**Comment by aeiras** [comment]
@rstjames you can use this ticket to update us on hootsuite account set up, using the group email. 
Currently trial under professional subscription. Thanks!

**Comment by rstjames** [comment]
@aeiras what do you want me to update? Do we have a login information? 
I have not been involved in the hootsuite setting up...

**Comment by aeiras** [comment]
thanks, @TetianaFTV, for pushing this activity forward. 
Can you look at the suggested description on this ticket, please?
 https://github.com/microprofile/microprofile-wg/issues/72

If you could follow there via comments, that would allow us to complete the translation work. 
The Spanish version of the document is almost completed, awaiting 1 item from Emerson. 

Thank you,

**Comment by aeiras** [comment]
With Ryan, this AM, we have tested the access and unblocked the issues. More updates, from Ryan as he sees fit!

**Comment by rstjames** [comment]
up and running. I am not sure how to get linkedin and facebook linked... Do we have the email and password for facebook? @aeiras

**Comment by aeiras** [comment]
@rstjames:

1. INFRA- Social Media Admins] Update for Linkedin [#509](https://github.com/eclipse/microprofile-marketing/issues/509)
2.  [INFRA- Social Media Admins] Facebook Update and Clean up[ #510](https://github.com/eclipse/microprofile-marketing/issues/510)

**Comment by rstjames** [comment]
linkedin and twitter are linked. still dealing with Facebook
